src/main/java/ua/edu/internship/interview/web/controller/SkillController.java

```diff
@@ -0,0 +1,30 @@
+package ua.edu.internship.interview.web.controller;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ua.edu.internship.interview.service.business.SkillService;
+import ua.edu.internship.interview.service.dto.SkillTreeDto;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChA5NGNlOTc2ZjUyNDMwYmRlEgpjaGVja3N0eWxl -->